src/lib.rs

```diff
@@ -5,6 +5,7 @@ mod device;
 pub mod frame;
 
 // Helper Methods
+pub use device::display::get_all_displays;
 pub use device::display::get_targets;
 pub use device::display::has_permission;
```

Comment:
imo we can skip exposing `get_all_displays` here. We should keep our API footprint small and let people call `get_targets` and then filter out what they need from there itself.

Reply:
Done